src/main/kotlin/racingcar/model/RandomNumber.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.model
+
+import kotlin.random.Random
+
+private const val RANDOM_RANGE = 10
+
+interface NumberGenerator {
+    fun generateNumber(): Int
+}
```

Comment:
```suggestion
fun interface NumberGenerator {
    fun generateNumber(): Int
}
```

이렇게 interface 앞에 fun을 붙이면 SAM으로 구현이 가능해집니다!